docs/en/advanced_config/flight_termination.md

```diff
@@ -69,6 +69,10 @@ For each AUX output to which a safety device is attached, where "n" is the PWM p
 
 Finally, set the `PWM_AUX_FAILn` and `PWM_MAIN_FAILn` PWM values for any motors.
 
+::: info
+Flight termination via ATS only works if `drivers/pwm_input` is included in the default build for your board. If not, you need to add it manually via [boardconfig](../hardware/porting_guide_config.md#px4-menuconfig-setup)
```

Comment:
Well, it has to be included in firmware but it could be in some other config that the default board.

```suggestion
Flight termination via ATS only works if `drivers/pwm_input` is included in the firmware for your board.
If not, you need to add it manually to your board configuration using [boardconfig](../hardware/porting_guide_config.md#px4-menuconfig-setup).
```